mne/io/boxy/boxy.py

```diff
@@ -50,73 +50,104 @@ class RawBOXY(BaseRaw):
     --------
     mne.io.Raw : Documentation of attribute and methods.
     """
-
+    
     @verbose
-    def __init__(self, fname, preload=False, verbose=None):
+    def __init__(self, fname, datatype=None, preload=False, verbose=None):
         from ...externals.pymatreader import read_mat
         from ...coreg import get_mni_fiducials, coregister_fiducials  # avoid circular import prob
         logger.info('Loading %s' % fname)
 
         # Check if required files exist and store names for later use
         files = dict()
-        keys = ('mtg', 'elp', 'tol', '001')
+        keys = ('mtg', 'elp', '*.[000-999]*')
+        print(fname)
         for key in keys:
-            files[key] = glob.glob('%s/*%s' % (fname, key))
+            if key == '*.[000-999]*':
+                files[key] = [glob.glob('%s/*%s' % (fname, key))]
+            else:
+                files[key] = glob.glob('%s/*%s' % (fname, key))
             if len(files[key]) != 1:
                 raise RuntimeError('Expect one %s file, got %d' %
                                    (key, len(files[key]),))
             files[key] = files[key][0]
-
+            
+        ###determine which data type to return###
+        if datatype in ['AC','DC','Ph']:
+            data_types = [datatype]
```

Comment:
what happens if people ask for ['AC', 'Ph']?
I think lets get them to pick one for now

Reply:
yeah right now it'll complain if you don't just pass a string (just 'AC' for example) but the rest of the code should be able to handle getting data for multiple types.

Reply:
ok, lets focus on use case where user only wants one for now to make naming simpler